### PR TITLE
Adding a Script for LazyLibrarian

### PR DESCRIPTION
**Pull request by MountyMapleSyrup**
Install script for LazyLibrarian. Tested it out as well as the update and it all appears to be working correctly. LazyLibrarian is installed from source code from their GitLab which is less than ideal but that is what their documentation says to do.

- [ ] New Script (Develop a new script or set of scripts that are fully functional and thoroughly tested)
- [ ] I have performed a self-review of my code, adhering to established codebase patterns and conventions.
- [ ] This change requires a documentation update 
-------^^^ not sure if this requires a documentation update, I think it would but uncertain. I can make changes if needed to update the documentation

### CODE REVIEW

**Comment by tteck** [comment]
I update `CHANGELOG.md` when it's added to the website, not before.

https://www.patorjk.com/software/taag/#p=display&f=Slant&t=LazyLibrarian

**Comment by tteck** [comment]
```
# Author: tteck
# Co-Author: MountyMapleSyrup (MountyMapleSyrup)
```

```
msg_info "Installing Dependencies"
$STD apt-get install -y curl
$STD apt-get install -y sudo
$STD apt-get install -y mc
$STD apt-get install -y git
msg_ok "Installed Dependencies"

msg_info "Installing Python3 Dependencies"
$STD apt-get install -y pip
$STD apt-get install -y python3-irc
$STD pip install jaraco.stream
$STD pip install python-Levenshtein
$STD pip install soupsieve
msg_ok "Installed Python3 Dependencies"
```

`systemctl -q daemon-reload` not needed

File: CHANGELOG.md
Comment (tteck) [inline CHANGELOG.md:1]: Remove
I update `CHANGELOG.md` when it's added to the website, not before.

File: ct/lazylibrarian.sh
Comment (tteck) [inline ct/lazylibrarian.sh:1]: https://www.patorjk.com/software/taag/#p=display&f=Slant&t=LazyLibrarian

```
# Author: tteck
# Co-Author: MountyMapleSyrup (MountyMapleSyrup)
```

File: install/lazylibrarian-install.sh
Comment (tteck) [inline install/lazylibrarian-install.sh:1]:
```
msg_info "Installing Dependencies"
$STD apt-get install -y curl
$STD apt-get install -y sudo
$STD apt-get install -y mc
$STD apt-get install -y git
msg_ok "Installed Dependencies"

msg_info "Installing Python3 Dependencies"
$STD apt-get install -y pip
$STD apt-get install -y python3-irc
$STD pip install jaraco.stream
$STD pip install python-Levenshtein
$STD pip install soupsieve
msg_ok "Installed Python3 Dependencies"
```

`systemctl -q daemon-reload` not needed

File: CHANGELOG.md
Comment (MountyMapleSyrup) [reply CHANGELOG.md:1]: updated

File: ct/lazylibrarian.sh
Comment (MountyMapleSyrup) [reply ct/lazylibrarian.sh:1]: updated

File: install/lazylibrarian-install.sh
Comment (MountyMapleSyrup) [reply install/lazylibrarian-install.sh:1]: updated

File: ct/lazylibrarian.sh
Comment (tteck) [reply ct/lazylibrarian.sh:1]: Missed
```
# Author: tteck
# Co-Author: MountyMapleSyrup (MountyMapleSyrup)
```

File: install/lazylibrarian-install.sh
Comment (tteck) [reply install/lazylibrarian-install.sh:1]: Missed
```
# Author: tteck
# Co-Author: MountyMapleSyrup (MountyMapleSyrup)
```
```
msg_info "Installing Dependencies"
$STD apt-get install -y curl
$STD apt-get install -y sudo
$STD apt-get install -y mc
$STD apt-get install -y git
msg_ok "Installed Dependencies"

msg_info "Installing Python3 Dependencies"
$STD apt-get install -y pip
$STD apt-get install -y python3-irc
$STD pip install jaraco.stream
$STD pip install python-Levenshtein
$STD pip install soupsieve
msg_ok "Installed Python3 Dependencies"
```

File: ct/lazylibrarian.sh
Comment (MountyMapleSyrup) [reply ct/lazylibrarian.sh:1]: updated in both files

File: install/lazylibrarian-install.sh
Comment (MountyMapleSyrup) [reply install/lazylibrarian-install.sh:1]: updated as well

Comment (tteck) [reply install/lazylibrarian-install.sh:1]: Did you?

Comment (MountyMapleSyrup) [reply install/lazylibrarian-install.sh:1]: i didnt.  but i believe now i have. sorry for confusion on my part.

**Comment by tteck** [comment]
Thanks

**Pull request merged**